Sources/Common/Store/SdkConfig.swift

```diff
@@ -66,13 +75,3 @@ public struct SdkConfig {
      */
     public var _sdkWrapperConfig: SdkWrapperConfig? // swiftlint:disable:this identifier_name
 }
-
-public protocol SdkConfigStore: AutoMockable {
-    var config: SdkConfig { get set }
-}
-
-// sourcery: InjectRegister = "SdkConfigStore"
-// sourcery: InjectSingleton
-public class InMemorySdkConfigStore: SdkConfigStore {
-    @Atomic public var config = SdkConfig()
-}
```

Comment:
No more need for a SDK store. We now store the SDKConfig instance in the dependency injection graph.